R/lazyframe-frame.R

```diff
@@ -1015,7 +1015,11 @@ lazyframe__reverse <- function() {
 lazyframe__drop_nulls <- function(subset = NULL) {
   wrap({
     if (!is.null(subset)) {
-      subset <- parse_into_list_of_expressions(!!!subset)
+      if (is_polars_selector(subset)) {
+        subset <- parse_into_list_of_expressions(!!!list(subset))
+      } else {
+        subset <- parse_into_list_of_expressions(!!!subset)
+      }
```

Comment:
Fix when a polars selector is passed. Without this we have an error "can't splice an environment"